src/core/viewrenderer/web/ViewRenderer.js

```diff
@@ -33,7 +33,7 @@ class ViewRenderer extends React.Component {
     }
 
     _getTransform() {
-        return 'translate(' + this.props.x + 'px,' + this.props.y + 'px)';
+        return 'translate(' + this.props.x + 'px,' + this.props.y + 'px) ' + (this.props.inverted ? this.props.isHorizontal ? "scaleX(-1)" : "scaleY(-1)" : "");
```

Comment:
Computing transform upfront might be better, will reduce the number of isHorizontal checks

Reply:
Lot of isHorizontal checks are creeping in, I'm thinking of ways to prevent that

Reply:
Agree